shared/actions/chat.js

```diff
@@ -298,8 +299,12 @@ function * _setupNewChatHandler (): SagaGenerator<any, any> {
 const followingSelector = (state: TypedState) => state.config.following
 
 function * _loadInbox (): SagaGenerator<any, any> {
-  // $ForceType
-  const inbox: GetInboxAndUnboxLocalRes = yield call(localGetInboxAndUnboxLocalRpcPromise, {params: {}})
+  const inbox: GetInboxAndUnboxLocalRes = ((yield call(localGetInboxAndUnboxLocalRpcPromise, {param: {
+    query: {
+      status: Object.keys(CommonConversationStatus).filter(k => !['ignored', 'blocked'].includes(k)).map(k => CommonConversationStatus[k]),
```

Comment:
I did this on purpose in case we add new types. we likely want them instead of them not being there until we explicitly add them